### ISSUE TITLE
Documentation enhancement

### ISSUE THREAD

**Issue by azmeuk** [cocos2d/bindings-generator#140]
The [documentation](https://github.com/cocos2d/bindings-generator/README.md) explains how to generates jsbindings, but it does not tell the proper way to integrate it into a cocos2d-js project.
- What steps should be followed?  
- Where manually written C++ files should be stored?  
- Where js and c++ generated files should be stored?  
- What CMakeList.txt files should be edited?

**Comment by Teivaz** [comment]
Yes. There is no clear way to integrate custom bindings to cocos. Maybe someone can introduce one?

**Comment by azmeuk** [comment]
I wrote a draft [on stack overflow](http://stackoverflow.com/questions/26762205/what-is-the-proper-way-to-set-up-a-jsbindings-class-with-cocos2d-js/).

**Comment by jgod** [comment]
There's fantastic functionality here, shame it's undocumented!
